### DISCUSSION TITLE
Is there a quality check for consistent translation?

### DISCUSSION THREAD

**Discussion by Nin83** [WeblateOrg/weblate#7570] [General]
Hi is there a quality check for consistent translations? Not just for entire strings but also single words or expressions included in longer strings?
Thanks!

**Comment by comradekingu** [comment]
https://docs.weblate.org/en/latest/admin/continuous.html?highlight=consistency#consistency-check
For single words there is https://docs.weblate.org/en/latest/user/glossary.html?highlight=glossary#managing-glossaries
I guess you could mark as terminology, and then do all forbidden possibilities and possible variants.

I can have a look manually if you want, because it is somewhat more involved than just getting the words right. :)
